ayanda/src/main/java/sintulabs/p2p/IBluetooth.java

```diff
@@ -0,0 +1,20 @@
+package sintulabs.p2p;
+
+import android.content.Intent;
+
+/**
+ * Created by sabzo on 1/21/18.
+ */
+
+public interface IBluetooth {
```

Comment:
This is not a common naming pattern in Java. Consider `Bluetooth.Listener` or `Bluetooth.Callback` instead.